hw2/03-form.html

```diff
@@ -12,8 +13,35 @@
     <script defer src="03-form.js"></script>
   </head>
   <body class="bg-dark">
-    <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
+    <form class="bg-light border rounded w-25 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">Contact Form</h1>
+      <div class="form-group">
+        <label for="inputemail">Name * </label>
+        <input type="name" class="form-control" id="inputname" required />
+        <label for="inputemail">Email * </label>
+        <input type="email" class="form-control" id="inputemail" required />
+        <label for="inputmessage">Submit your message: </label>
+        <textarea type="name" class="form-control" id="inputmessage" rows="3"></textarea>
+        <input type="checkbox" id="check" />
+        <label for="check">Sign up for the newsletter</label>
+      </div>
+      <div class="row">
+        <input
```

Comment:
Both of the input buttons could use some margins to be inline with the rest of the form.